### PR TITLE
RELATED: RAIL-3428 - Rename refactor in plugin SPI

### PR DESCRIPTION
**Pull request by lupko**
-  insightRendering -> insightWidgets()
-  kpiRendering -> kpiWidgets()
-  widgets -> customWidgets()

This will grow better into the future where we may want to add additional customizations on how the insight or KPI widgets & their different sub-components get rendered (e.g. we may want to enhance insightWidgets() with funs to add custom content into button bar, customize how title is rendered. customize before/after/left/right components)

RELATED: RAIL-3428

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui-dashboard/src/plugins/customizationApis/customizationBuilder.ts [hunk 1]
```diff
@@ -32,15 +32,15 @@ export class DashboardCustomizationBuilder implements IDashboardCustomizer {
...
         return this.insightCustomizer;
     };
 
-    public kpiRendering = (): IDashboardKpiCustomizer => {
+    public kpiWidgets = (): IDashboardKpiCustomizer => {
         return this.kpiCustomizer;
     };
 
-    public widgets = (): IDashboardWidgetCustomizer => {
+    public customWidgets = (): IDashboardWidgetCustomizer => {
```
Comment (no23reason) [inline libs/sdk-ui-dashboard/src/plugins/customizationApis/customizationBuilder.ts:43]: Is this really only custom widgets? It is also insight and kpi widgets too, no?

**Review by no23reason — APPROVED** [review]
LGTM

File: libs/sdk-ui-dashboard/src/plugins/customizationApis/customizationBuilder.ts (hunk 1 above)
Comment (no23reason) [reply libs/sdk-ui-dashboard/src/plugins/customizationApis/customizationBuilder.ts:43]: nvm, it is only custom widgets after all.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1918

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1662/ : SUCCESS in 1m 13s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1942/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/739/ : SUCCESS in 15m 01s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7562/ : SUCCESS in 19m 08s

**Pull request merged**